templateflow/tests/test_api.py

```diff
@@ -0,0 +1,29 @@
+from .. import api
+
+
```

Comment:
Please add tests (maybe parametrized?):
- With only URLs (no dois)
- Mixing urls and dois